src/org/daisy/dotify/formatter/impl/VolumeProvider.java

```diff
@@ -295,7 +295,7 @@ private SectionBuilder updateVolumeContents(int volumeNumber, ArrayList<AnchorDa
             }
             return sb;
         } catch (PaginatorException e) {
-            return null;
+            throw new RuntimeException(e);
```

Comment:
Why not simply remove the `try` .. `catch` ? This seems a bit tricky to me.

Reply:
I guess we could expose PaginatorException in some more places. But what is the point really? Then it still needs to be caught further down and converted to some another kind of exception because we don't expose PaginatorException in the [formatter API](https://mtmse.github.io/dotify.api/latest/javadoc/index.html). Check the code, there are several other places where a PaginatorException is converted to a RuntimeException.